deploy/helm/sumologic/values.yaml

```diff
@@ -4125,6 +4141,217 @@ metadata:
       ## To use maxUnavailable, set minAvailable to null and uncomment the below:
       # maxUnavailable: 1
 
+## Configure log collection with Otelcol
+## This is currently highly experimental and not recommended unless you really know what you're doing.
+## In particular, this configuration currently only covers container logs, and only supports docker-shim.
+## Unlike the fluent-bit configuration, it also reads logs from /var/log/pods instead of /var/log/containers
+## which is consistent with CRI, but may possibly cause issues on older K8s versions.
+## This is an alpha feature, and may change in the near future.
+otellogs:
+  enabled: false
+
+  ## Configure image for Opentelemetry Collector
+  image:
+    repository: public.ecr.aws/sumologic/sumologic-otel-collector
+    tag: 0.0.45-beta.0
+    pullPolicy: IfNotPresent
+
+  logLevel: info
+  config:
```

Comment:
Just a thought: since there's not all that much that we'd like to allow folks to customize in here I believe it would be much simpler (not littering `values.yaml` and not complicating it even further) to put this into a config map and then expose what will end up necessary to be exposed.

WDYT ?

Reply:
Hm, since we're going to expose OTC configuration for metadata anyway, I don't think this causes any issues. We allow it for Fluent-Bit, whose configuration is less user-friendly. And users will want to modify this configuration in the future, to set up multiline logs correctly, if nothing else. I think trying to hide the details in this case would ultimately create more complexity for both us and the users.

Reply:
🤔 Perhaps this makes sense w.r.t to multiline logs.